src/peft/tuners/lora/model.py

```diff
@@ -194,6 +195,13 @@ def _create_and_replace(
                 new_module.requires_grad_(False)
             self._replace_module(parent, target_name, new_module, target)
 
+        if lora_config.use_rslora:
```

Comment:
I think here is not the right place to control the scaling, as this leads to spreading the initialization of the LoRA parameters into different parts of the code. Instead, `update_layer`, `update_layer_conv2d`, and `update_layer_embedding` in `tuners/lora/layer.py` should be adjusted, since that's where we set the scale initially. This also requires updating the `__init__` method to accept the new argument, as well as the `kwargs` variable [here](https://github.com/huggingface/peft/blob/0f1e9091cc975eb5458cc163bf1843a34fb42b76/src/peft/tuners/lora/model.py#L151-L157).